modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -63,6 +63,86 @@ definition {
 		}
 	}
 
+	@description = "LPS-180358 - Verify that the status follow the correct workflow after a finance user request more info"
+	@priority = 5
+	test CanFinanceUserVerifyStatusAfterRequestMoreInfo {
+		task ("Given that a new organization was created with employee user") {
+			User.logoutAndLoginPG(
```

Comment:
I think you can change this task for: `"Given There is a created Organization by an Employee user" `